src/google/protobuf/compiler/java/java_helpers.h

```diff
@@ -154,7 +154,7 @@ std::string GetOneofStoredType(const FieldDescriptor* field);
 
 // Whether we should generate multiple java files for messages.
 inline bool MultipleJavaFiles(const FileDescriptor* descriptor,
-                              bool immutable) {
```

Comment:
@georgthegreat Thanks for the cleanup! I think this should work except that our internal version of `MultipleJavaFiles` does use both parameters, so for this particular one I think it would be better to leave the parameter name but add something like `(void) immutable;`